### ISSUE TITLE
Fix broken unittests

### ISSUE THREAD

**Issue by pimjansen** [Azure-OSS/azure-storage-php#13]
All the unittests are broken since they ran without Azure pipelines with keys in env. We should find a proper way to run those again via Github actions

**Comment by lrljoe** [comment]
Should just be able to set those with some org secrets

**Comment by pimjansen** [comment]
> Should just be able to set those with some org secrets

Thats correct for the functional integration tests but the unittests are also pretty broken. I updated the CI to run only the unittests atm
